Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml

```diff
@@ -0,0 +1,84 @@
+﻿id: f7abe9c1-1e6c-4317-b907-25769e7764c5
+name: Digital Shadows Incident Creation for exclude-app
+description: ''
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalShadows
+    dataTypes: 
+      - DigitalShadows_CL
+queryFrequency: 5m
+queryPeriod: 6m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: []
+query: let DSSearchLight_view  = view () {
```

Comment:
For the sake of uniformity, can we please modify the query section. Here is a detection query for reference:
https://github.com/Azure/Azure-Sentinel/blob/master/Detections/AuditLogs/BulkChangestoPrivilegedAccountPermissions.yaml

Please look at things like:
* Actual query should start from next line of the query field.
*  Pipe operator | should be in the new line.